inference-engine/tests/functional/plugin/cpu/single_layer_tests/adaptive_pooling.cpp

```diff
@@ -81,6 +81,7 @@ class AdaPoolLayerCPUTest : public testing::WithParamInterface<AdaPoolLayerCPUTe
         threshold = 1e-2;
         function = (mode == "max" ? std::make_shared<ngraph::Function>(adapoolMax->outputs(), params, "AdaPoolMax") :
                     std::make_shared<ngraph::Function>(adapoolAvg->outputs(), params, "AdaPoolAvg"));
+        functionRefs = ngraph::clone_function(*function);
```

Comment:
why do we need it?

Reply:
discussed offline. Used for interpreter case function was transformed

Reply:
@ilya-lavrenov Yes, sure, wip